FirstProject/css/feedback.css

```diff
@@ -89,15 +78,16 @@ h1.interview_text_area {
     }
 
     .left_image {
-        float: left!important;
+        width: 100%;
     }
 
     header form {
         margin-left: 0!important;
     }
 
     .header_form_section {
-        width: 94%!important;
+        width: 87%;
+        margin: 0 40px auto auto;
```

Comment:
I set the width to 87% and I set the margin-top: 0;
margin-right: 40px; and margin-bottom && left: auto.